app/models/concerns/spotlight/solr_document.rb

```diff
@@ -42,16 +42,20 @@ def update current_exhibit, new_attributes
       end
     end
 
-    def reindex
+    def reindex merge = false
       # no-op reindex implementation
     end
 
+    def update_index data = [], merge = false
+      # no-op update index implementation
+    end
+
     def sidecar exhibit
       sidecars.find_or_initialize_by exhibit: exhibit
     end
 
     def to_solr
-      { id: id }.reverse_merge(sidecars.inject({}) { |result, sidecar| result.merge(sidecar.to_solr) }).merge(tags_to_solr)
```

Comment:
Thanks for doing `id` correctly in `Upload`. Hopefully I've fixed all the places I did it wrong before..